test/e2e/fixtures/lint/file-with-prettier-lint-errors.ts

```diff
@@ -1,5 +1,11 @@
-export const foo  = (   ) => 
-!! ('bar')
-;
+export const foo
```

Comment:
why changing this file?

Reply:
Ah this caused an test failure during linting, so to fix the tests I changed it back to faulty file.